tests/build/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
build folder and no build team members in owners file?

Reply:
Hi @Michkov , now only add ec team as approvers in this pr, since we have ec e2e test cases in e2e-tests/tests/build/chains.go. Feel free to add more build team members. We are also discussing the possibility to move ec e2e test cases out of build.